### PR TITLE
apt-packages integration test

### PR DESCRIPTION
**Pull request by tiff**
Tests should fail for now until experimental-platform/platform-configure-script#23 is done.
See https://github.com/experimental-platform/nodejs-hello-world/tree/example_aptfile for test case

### CODE REVIEW

**Comment by hinnerk** [comment]
The build fails, because the `Dockerfile` tries to `ADD run_tests.sh`:

```
Step 9 : ADD run_tests.sh /run_tests.sh

docker build -t test-integration-nodejs:latest . returned exit code 1

INFO[0095] run_tests.sh: no such file or directory
```

**Comment by hinnerk** [comment]
Apparently node.js is quite slow and might need a `CHECKS` file:

```
remote: -----> Deploying node-js-apt-packages...        
remote: -----> DOKKU_SCALE not found in app image. Defaulting to a single web process        
remote: -----> New DOKKU_SCALE file generated        
remote: =====> web=1        
remote: -----> Running pre-flight checks        
remote:        For more efficient zero downtime deployments, create a file CHECKS.        
remote:        See http://progrium.viewdocs.io/dokku/checks-examples.md for examples        
remote:        CHECKS file not found in container: Running simple container check...        
remote: -----> Waiting for 10 seconds ...        
remote: App container failed to start!!        
remote: =====> node-js-apt-packages container output:        
remote:        Detected 512 MB available memory, 512 MB limit per process (WEB_MEMORY)        
remote:        Recommending WEB_CONCURRENCY=1        
remote:        Node app is running at localhost:5000        
remote: =====> end node-js-apt-packages container output        
remote: /usr/local/bin/dokku: line 49:  1409 Killed                  dokku deploy "$APP"        
To ssh://dokku@104.131.227.123:8022/node-js-apt-packages
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'ssh://dokku@104.131.227.123:8022/node-js-apt-packages'
```

**Comment by tiff** [comment]
This rather looks like a memory issue to me because of `Detected 512 MB available memory, 512 MB limit per process (WEB_MEMORY)`

File: run_apt_packages_test.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+echo -e "\nCLONING repo\n"
+git clone https://github.com/experimental-platform/nodejs-hello-world.git
+cd nodejs-hello-world/
+echo -e "\nCHECKING OUT branch\n"
+git checkout example_aptfile
+git config user.email "platform@protonet.info"
+git config user.name "Protonet Integration Test node.js"
+# http://progrium.viewdocs.io/dokku/checks-examples.md
+echo -e "WAIT=10\nATTEMPTS=20\n/ GraphicsMagick" > CHECKS
```
Comment (tiff) [inline run_apt_packages_test.sh:19]: @hinnerk I think this is the reason why the tests are failing: `GraphicsMagick` is only returned when the apt-get worked. So this should be fine once we have the dokku plugin in place

**Pull request merged**